contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -2695,12 +2721,18 @@ execute_bulk_load_insert(int ncol, int nrow, Oid *argtypes,
 		for (int i = 0; i < nrow; i++)
 		{
 			for (int j = 0; j < ncol; j++)
-				appendStringInfo(bindParams, ",$%d", count++);
-
+			{
+				/* If Defaults is set then we need to insert default value for this index. */
+				if (Defaults[i * ncol + j])
+					appendStringInfo(bindParams, ",DEFAULT");
+				else
+					appendStringInfo(bindParams, ",$%d", count++);
```

Comment:
Have you tested this with all the datatype including money, varbinary?

Reply:
Tests were added since DAY 1 of this code being pushed. (The first commit where we supported Insert bulk -- Bulk load request)